tools/ci_build/github/linux/docker/scripts/requirements.txt

```diff
@@ -4,7 +4,7 @@ mypy
 pytest
 setuptools>=41.4.0
 wheel
-git+http://github.com/onnx/onnx.git@971632833036c576cf95499291f689f8bb3519e1#egg=onnx
+onnx==1.9.0
```

Comment:
Is this change needed? If yes, could you please apply the same change to https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt ? We should use the same ONNX version across all the pipelines. Also, please help us update C:\src\onnxruntime\cmake\external\onnx to the same commit id.

Reply:
I'll try running the entire CI without this change in the hopes that the change is not required. If that fails, I will update all the other files you have mentioned as well.

Reply:
Thank you

Reply:
Leaving it unchanged seems to have worked. Thanks.